client/starwhale/core/runtime/model.py

```diff
@@ -752,12 +752,15 @@ def list(
     @classmethod
     def quickstart_from_uri(
         cls,
-        workdir: Path,
+        workdir: t.Union[Path, str],
```

Comment:
is it always a PATH type? because in 'starwhale.core.runtime.cli._quickstart_from_uri'  has deal with 'p_workdir = Path(workdir).absolute()'

Reply:
Core model's function only concerns the handlable types which don't know the client's behavior.

Reply:
ok